src/dmd/json.d

```diff
@@ -999,6 +999,8 @@ public:
 extern (C++) void json_generate(OutBuffer* buf, Modules* modules)
 {
     scope ToJsonVisitor json = new ToJsonVisitor(buf);
```

Comment:
Why I didn't add this in in the destructor:
`scope` with classes is planned to be deprecated. While that's controversial (see e.g. https://github.com/dlang/dlang.org/pull/2027), imho it's best not to rely on a feature that might will be removed soon.